src/Book/SearchBook/SearchBookTable.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {useEffect} from 'react';
+import GenericTable, {Column} from "../../common/GenericTable/GenericTable";
+import {get} from "../../utils/http";
+import {Book} from "../Book";
+import "./SearchBookTable.css";
+import ActivateOrDeactivateButton from "./ActivateOrDeactivateButton";
+
+type Props = {
+    isAdmin: boolean,
+    openBookDetails: (id: number) => void,
+}
+
+const constColumns: Column[] = [
+    {
+        header: 'Titulo',
+        accessor: 'title'
+    },
+    {
+        header: 'Autor',
+        accessor: 'author'
+    },
+    {
+        header: 'Editorial',
+        accessor: 'publisher'
+    },
+]
+
+const SearchBookTable = (props: Props) => {
+    const columns: Column[] =
+        [
+            ...constColumns,
+            {
+                header: 'Acciones',
+                component: props.isAdmin ?
+                    (row => (
+                        <div className={'admin-search-actions'}>
+                            <ActivateOrDeactivateButton defaultValue={row.active} id={row.id}/>
```

Comment:
[AWAIT TO CONFIRMATION TO MODIFY]
This should follow the same logic as the details modal. 
It should be only 1 "ActivateDeactivateModal" (preferable inside the SEARCH section as commented before for the DETAILS) and then on the `SearchBookTable` you open that modal.

This change will avoid rendering 10 uneccessary modals on the SearchBookTable.

Reply:
Decided not to do it for this version. Closing revision.